modules/core/packer/roles/nomad/files/templates/syslog.hcl

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
This file is HCL. Don't need to be in JSON format. Don't need the quotes in keys and the braces.

```hcl
enable_syslog = {{ enable_syslog | to_json }}
```

Reply:
Good catch, fixed